httpstan/views.py

```diff
@@ -285,7 +286,10 @@ def _services_call_done(
         if exc:
             # e.g., "hmc_nuts_diag_e_adapt_wrapper() got an unexpected keyword argument, ..."
             # e.g., "Found negative dimension size in variable declaration"
-            message, status = f"Error calling services function: `{exc}`", 400
+            message, status = (
+                f"Exception during call to services function: `{repr(exc)}`, traceback: `{traceback.format_tb(exc.__traceback__)}`",
```

Comment:
How is the output from this combination?

Reply:
It doesn't add much information. The key thing is the exception type. Initialization failure is a ValueError but I think other errors should be different.